client/views/room/Header/Omnichannel/QuickActions/QuickActions.tsx

```diff
@@ -35,7 +36,7 @@ import { QuickActionsActionConfig, QuickActionsEnum } from '../../../lib/QuickAc
 import { QuickActionsContext } from '../../../lib/QuickActions/QuickActionsContext';
 
 type QuickActionsProps = {
-	room: IOmnichannelRoom;
+	room: IOmnichannelRoom & IOmnichannelSubscription;
```

Comment:
Why do we need to support `IOmnichannelSubscription` interface here given that `useOmnichannelRoom` always return just `IRoom` ?